src/components/TodoList/TodoList.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import axios from "axios";
+import TodoItem from "./TodoItem.jsx"
+
+const TodoList = (props = (todos, isLoading, setIsLoading)) => {
+    // TODO: Make interface for todos object
+    return (
+        <div className="bg-purple-300">
+            {props.isLoading ?
```

Comment:
Having so many conditionals is hard to read.
A better way to do it is: 
```
if(isLoading){
 return <Loader/>
}

return (
 <>
 <p>Todo list:</p>
 {todos.length ? (...) : (...)}
</>
)
```

That way you skip one level of nesting. Or even if you make a more complex fallback:

```
if(isLoading){
 return <Loader/>
}

if(todos.length === 0){
 return <Fallback/>
}

return (
 <div>
 <p>Todo list:</p>
 {todos.map(x => x)}
</div>
)
```